R/apply_row_grp_plan.R

```diff
@@ -96,6 +96,15 @@ apply_row_grp_struct <- function(.data, row_grp_struct_list, group, label = NULL
 #' @importFrom rlang !!!
 apply_row_grp_lbl <- function(.data, element_row_grp_loc, group, label = NULL, ...){
 
+  # store values of label column
+  lbl_col <- rlang::eval_tidy(label, .data)
```

Comment:
please move this to @importFrom rlang eval_tidy, and remove the rlang::

Reply:
see latest commit :)